infra/scripts/replace-running-components.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+BASE_DIR=$(dirname $(readlink -f "$BASH_SOURCE"))
+VIC_DIR=$(dirname $(readlink -f $BASE_DIR/..))
+
+# Grab vSphere's thumbprint for calling vic-machine
+function get-thumbprint () {
+    govc about.cert | grep "SHA-1 Thumbprint" | awk '{print $NF}'
```

Comment:
`govc about.cert -k -thumbprint` gives you the thumbprint undercoated so you don't need the grep/awk.

Reply:
oh nice!

Reply:
still prints the IP so I do need awk, but not sed